Demo/Ser_Des/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+project(Ser_Des)
+
+add_executable(Serializaion_test_1 test_case_1.cpp)
```

Comment:
Don't create an executable for each test. Combine them into a single executable.